### PR TITLE
Add help button to sliceviewer

### LINKED ISSUE #31262: SliceViewer missing Help button
As noted in Manual Testing #31111 

### Expected behavior
SliceViewer GUI should have a ? button linking to relevant help

### Actual behavior
There is no ? button in SliceViewer

### Steps to reproduce the behavior
Load a file and then open SliceViewer.

### Platforms affected
Windows 10

### PR DESCRIPTION
**Pull request by joseph-torsney**
Adds a help button to the sliceviewer

**To test:**

1. Open sliceviewer
2. Click the help button in the bottom left
3. Check that it opens the correct page in the mantid docs

Fixes #31262. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by RichardWaiteSTFC** [comment]
I get a missing link - but presumably I need to build docs?
I did have one question though - is it possible to get rid of extraneous spacers/dividers?
image

**Comment by RichardWaiteSTFC** [comment]
Built the `docs-html` OK, but when I build `docs-qthelp` i get the following error in VS
```
Error	MSB6006	"cmd.exe" exited with code -1.	docs-qthelp	C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets	231
```
And the help page is still not working (although the link looks OK to me)

**Comment by thomashampson** [comment]
I rebuilt the Windows job and built the package to test the button and can confirm it works as expected.

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 1]
```diff
@@ -23,6 +23,8 @@ New and Improved
 - Added tooltips to all the widgets in the Slice Viewer. Please contact the developers if any are missing.
 - Script editor tab completion and call tip support for Numpy 1.21
 - The visibility of a component parameter in the Pick tab of the InstrumentViewer is now steered by the 'visible' atrribute of a parameter in IPF
+- Improved cross-platform theming by using the same icon on all platforms for closing editor tabs.
```
Comment (thomashampson) [inline docs/source/release/v6.2.0/mantidworkbench.rst:26]: This change was reverted, so the line shouldn't be here.

**Review by thomashampson — CHANGES_REQUESTED** [review]

**Review by thomashampson — APPROVED** [review]

**Comment by AnthonyLim23** [comment]
@mantidproject/gatekeepers could we please get this merged?

**Pull request merged**